app/js/components/LabResultsList.jsx

```diff
@@ -289,10 +289,6 @@ export class LabResultsList extends PureComponent {
                 defaultMessage="Date From: " />
             )}
             defaultDate={moment(labResultListFilters.dateFromField).format() || moment().subtract(8, 'days').format()}
-            formControlStyle={{
-              marginRight: '5px',
-              width: '105px',
-            }}
```

Comment:
Unrelated; this was present in order to make the date input box too small for its contents.